### ISSUE TITLE
What's the status of this project?

### ISSUE THREAD

**Issue by akefirad** [televator-apps/vimari#290]
Are there any active development? Any plan for bug fixes? Do you need a hand? I was wondering what is the status of this project. Thanks.

**Comment by nstylo** [comment]
I just switched to OSX and I was looking for a Vimium alternative. This project unfortunately seems unmaintained. Perhaps some people are interested in forking this project. I have no experience with Swift but I am good with JS.

**Comment by nikivdev** [comment]
Yea I have been meaning to do this. I want to understand code in depth to make some changes. Didn't get the time yet.

**Comment by nstylo** [comment]
Yep time seems to be my constraining factor as well at this point. I am keeping this project in mind as I am heavily using vim bindings throughout my system and I Id like to have them work in my browser as well.

**Comment by dzirtusss** [comment]
For those who might be interested, I've recently made public an alternative project for Safari - https://github.com/dzirtusss/vifari

It is not a browser extension like Vimium or Vimari, but uses macos accessabiliti api. From user perspective it feels almost the same. Even more, as it is pure lua, it is much easier to do something with it w/o any need for Swift/XCode/Apple/etc.

P.S. As well thanks to all the creators and mainteiners of Vimari. It was great project, and big pity it is not supported anymore. Was using it for some time and it was very helpful. Thanks.
